pytorch/edgeml_pytorch/graph/rnn.py

```diff
@@ -967,78 +967,142 @@ class BaseRNN(nn.Module):
     [batchSize, timeSteps, inputDims]
     '''
 
-    def __init__(self, cell: RNNCell, batch_first=False):
+    def __init__(self, cell: RNNCell, batch_first=False, cell_reverse: RNNCell=None, bidirectional=False):
```

Comment:
What is this RNNCell=None? I don't know what it does. Why should there be an  RNNCell for this argument?

Reply:
If the bidirectional pass doesn't share weights this passes an extra initialized RNNCell for doing the backward pass.

Reply:
Should I resolve this conversation ? @oindrilasaha